plenary/plenary-37.yaml

```diff
@@ -0,0 +1,839 @@
+metadata:
+  title: Resolutions of the 37th plenary meeting of ISO/TC 154
+  date: 2018-10-11
```

Comment:
In the original document, there are two dates indicated in the following form: `2018-10-11/12`. 
However, as you can see in the code, I put the first date (`2018-10-11`). But what would be the correct setting in this case?

Reply:
I think we should put:

```yaml
  date: 
    start: 2018-10-11
    end: 2018-10-12
```

Reply:
Ok